vllm/model_executor/layers/mamba/linear_attn.py

```diff
@@ -219,6 +218,8 @@ def __init__(
             raise ValueError(f"Duplicate layer name: {prefix}")
         compilation_config.static_forward_context[prefix] = self
 
+        self.mamba_attn_backend = get_mamba_attn_backend(self.mamba_type)
```

Comment:
`self.mamba_attn_backend` is only used by `get_attn_backend`, why not let `get_attn_backend` call `get_mamba_attn_backend` directly? this `self.mamba_attn_backend` is looks unnecessary